<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>547</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>666</width>
    <height>547</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>666</width>
    <height>547</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>2</width>
        <height>500</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>2</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>81</width>
        <height>2</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>2</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>2</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>5</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>2</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>90</y>
        <width>241</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/akainu.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>360</y>
        <width>661</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/grass.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>241</width>
        <height>301</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/ace-2.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="fundoPerna">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>71</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/fundoPreto.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="fundoPerna2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>71</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/fundoPreto.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="FundoBraco2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>120</y>
        <width>71</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/fundoPreto.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="fundoBraco">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>71</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/fundoPreto.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="fundoTorco">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>71</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/fundoPreto.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="fundoCabeca">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/fundoPreto.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="fundoAkainu">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>90</y>
        <width>251</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/fundoPreto.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;MS UI Gothic&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>HANGMAN GAME</string>
      </property>
     </widget>
     <widget class="QLabel" name="dica">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>311</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 50pt &quot;MS UI Gothic&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="input">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>370</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="chutar">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>380</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QLabel" name="resp">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>410</y>
        <width>311</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 50pt &quot;MS UI Gothic&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>line_4</zorder>
     <zorder>label_3</zorder>
     <zorder>fundoPerna</zorder>
     <zorder>label_2</zorder>
     <zorder>line_3</zorder>
     <zorder>fundoPerna2</zorder>
     <zorder>FundoBraco2</zorder>
     <zorder>fundoBraco</zorder>
     <zorder>fundoTorco</zorder>
     <zorder>fundoCabeca</zorder>
     <zorder>fundoAkainu</zorder>
     <zorder>line</zorder>
     <zorder>label</zorder>
     <zorder>label_4</zorder>
     <zorder>dica</zorder>
     <zorder>input</zorder>
     <zorder>chutar</zorder>
     <zorder>resp</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
